sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -393,7 +398,59 @@ async Task Step (int msg_id, StepKind kind, CancellationToken token)
 			await SendCommand ("Debugger.resume", new JObject (), token);
 		}
 
-		async Task GetScopeProperties (int msg_id, int scope_id, CancellationToken token)
+        async Task GetObjectProperties(int msg_id, int object_id, CancellationToken token)
```

Comment:
@thaystg It looks like the indention formatting is off.

Reply:
You are completely right. I think that now it's correct.